packages/frontend/components/scaffold-eth/Balance.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { useBalance } from "wagmi";
+
+type BalanceProps = {
+  address: string;
+  price: number;
+  minimized?: boolean;
+  wrapperClasses?: string;
+};
+/**----------------------
+ * Display balance of an address
+ * ---------------------*/
+
+export default function Balance({ address, price, minimized, wrapperClasses }: BalanceProps) {
+  const [isEthBalance, setIsEthBalance] = useState<boolean>(true);
+
+  const {
+    data: balanceData,
+    isError,
+    isLoading,
+  } = useBalance({
+    addressOrName: address,
+  });
```

Comment:
This doesn't refresh and we want it to (like in SE-1)

We need to explore wagmi options for this hook: watch / cache time / stale time.

Reply:
have used these options. 
     watch: true,
    cacheTime: 5_000,